.github/workflows/zombienet.yml

```diff
@@ -12,15 +12,15 @@ jobs:
       - uses: actions/checkout@v2
       - name: build
         run: |
-          yarn install --frozen-lockfile
```

Comment:
What's the difference?

Reply:
There was notice while using it tha frozen lockfile is now deprecated and immutable should be used instead